example/config.yaml

```diff
@@ -14,20 +14,18 @@ startClock: 2019-01-01T00:00:00+09:00
 # Optional (default: same as tick)
 metricsTick: 60
 
-# Metrics of simulated kubernetes cluster is written to a file at this path.
+# Metrics of simulated kubernetes cluster is written
+# to a file at this path or standard out if not specified.
```

Comment:
> standard out if not specfiied

Could you please update this comment?

After this line is fixed, I will merge this PR. Thank you!